tests/tx/test_genesis.py

```diff
@@ -20,39 +20,42 @@ def get_genesis_output():
 
 
 class GenesisTest(unittest.TestCase):
-    def test_pow(self):
-        genesis = get_genesis_transactions(None)
+    def setUp(self):
+        super().setUp()
+        self.storage = TransactionMemoryStorage()
 
+    def test_pow(self):
+        genesis = self.storage.get_all_genesis()
```

Comment:
You could have this line on the `setUp` method, so you don't repeat it in all the tests.

Reply:
They are independent test but it happened that all of them need the genesis. We have plenty of things in the `setUp` of other tests because all tests used those things but there are some new tests that doesn't. But I can move there if we agree that's the best place to be.

Reply:
I am okay keeping it the way it is. The idea was just to prevent code duplication.

Reply:
I mean, `setUp` is just to make it practical, we could have another `TestCase`, but it's just so few tests that it's maybe not worth the line count increase.